### DISCUSSION TITLE
How can I change/save order of fields in data grid ?

### DISCUSSION THREAD

**Discussion by PetroGromovo** [dbeaver/dbeaver#14805] [Q&A]
Hello,
I changed structure of my mysql table ( I changed order of many fields)
but opening this table with dbeaver 21.2 I see that all fields in datagrid are the same order I had before...
Looks like old settings were saved... Where can I reset/change these settings?

Thanks in advance!

**Comment by uslss** [comment]
@PetroGromovo do you save changes using Save filter settings button?
![image](https://user-images.githubusercontent.com/31996417/140022481-e8e5d86e-8c31-40db-930a-26c177e53311.png)

**Comment by PetroGromovo** [comment]
Not sure, maybe yes. Could you please explain how this functionality work and how can I clear it to default?

**Comment by uslss** [comment]
sorry for the late response.
You can read about it [here](https://github.com/dbeaver/dbeaver/wiki/Data-Editor)
